Formula/php-cs-fixer.rb

```diff
@@ -1,18 +1,27 @@
 class PhpCsFixer < Formula
   desc "Tool to automatically fix PHP coding standards issues"
   homepage "https://cs.symfony.com/"
-  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v3.3.2/php-cs-fixer.phar"
-  sha256 "8a7cb6fcbf916f01d4b423b7850b4401bc687275011b6f437322679a4fc4613f"
+  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/archive/refs/tags/v3.3.2.tar.gz"
+  sha256 "12d4109a589c730f34591862d016eb058432a010e7f125a4f5be7efb5ffff9c3"
   license "MIT"
+  revision 1
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "c9979da8cbb781fd78fd4a79d36a4be3fd0b8e2bbdff1e89b66e930a0452f0eb"
   end
 
+  depends_on "composer" => :build
   depends_on "php"
 
   def install
-    bin.install "php-cs-fixer.phar" => "php-cs-fixer"
+    system "composer", "config", "platform.php", "8.0.13"
+    system "composer", "update"
+
+    inreplace "php-cs-fixer", "#!/usr/bin/env php", "#!#{Formula["php"].opt_bin}/php"
```

Comment:
What if you keep the old formula, but create a wrapper that executes the `php-cs-fixer.phar` with the `$(brew --prefix)/opt/php@8.0/bin/php` binary?

Reply:
🤔 That could work as well and would be less invasive, I suppose. I'll try that.

Reply:
see #89978

I think, I like that version a lot better. Thanks for the suggestion. 🙂